include/z64cutscene.h

```diff
@@ -77,30 +85,77 @@ typedef struct {
     /* 0xA */ u16 textId2;
 } CsCmdTextbox; // size = 0xC
 
+typedef enum {
+    /*      0 */ CS_TEXTBOX_TYPE_DEFAULT,
+    /*      1 */ CS_TEXTBOX_TYPE_1,
+    /*      2 */ CS_TEXTBOX_TYPE_LEARN_SONG,
+    /*      3 */ CS_TEXTBOX_TYPE_3,
+    /*      4 */ CS_TEXTBOX_TYPE_BOSSES_REMAINS,
+    /*      5 */ CS_TEXTBOX_TYPE_ALL_NORMAL_MASKS,
+    /* 0xFFFF */ CS_TEXTBOX_TYPE_NONE = 0xFFFF
+} CsTextboxType;
+
 typedef struct {
-    /* 0x0 */ u16 unk0;
+    /* 0x0 */ u16 type;
     /* 0x2 */ u16 startFrame;
     /* 0x4 */ u16 endFrame;
     /* 0x6 */ u8 unk6;
     /* 0x7 */ u8 unk7;
     /* 0x8 */ u8 unk8;
     /* 0x9 */ UNK_TYPE1 pad9[0x3];
-} CsCmdUnk190; // size = 0xC
-
-typedef struct {
-    /* 0x0 */ UNK_TYPE4 unk0;
-    /* 0x4 */ UNK_TYPE4 unk4;
-} CsCmdUnk5A; // size = 0x8
+} CsRumble; // size = 0xC
```

Comment:
This change would be inconsistent with the names of the other cutscenes structs.
But at the same time, the name `CsRumble` would be inconsistent too, it should be `CsCmdRumble`.
I think I prefer to name the structs `CsCmd*` instead of `CutsceneCmd*` or `CutsceneCommand*`